admin/spec/components/previews/solidus_admin/ui/search_panel/component_preview/overview.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="mb-8">
+  <h6 class="text-gray-500 mb-3 mt-0">
+    Default
+  </h6>
+
+  <script>
+    window.addEventListener('<%= current_component.stimulus_id %>:submit', (event) => {
+      debugger
+      alert(`selected: ${event.detail.resultTarget.innerText}`)
+    })
+    window.addEventListener('<%= current_component.stimulus_id %>:search', (event) => {
+      console.log('Search', event.detail)
```

Comment:
Is this console log necessary?

Reply:
No, I thought it was ok to use it in lookbook to show off the provided data, I'll remove it